x-pack/plugins/security_solution/public/management/pages/policy/view/policy_details.tsx

```diff
@@ -78,23 +74,23 @@ export const PolicyDetails = React.memo(() => {
   const pageBody: React.ReactNode = useMemo(() => {
     if (policyApiError) {
       return (
-        <EuiPageTemplate template="centeredContent">
+        <EuiPageSection grow={false} alignment="center">
           <EuiCallOut color="danger" title={policyApiError?.error}>
             <span data-test-subj="policyDetailsIdNotFoundMessage">{policyApiError?.message}</span>
           </EuiCallOut>
-        </EuiPageTemplate>
+        </EuiPageSection>
       );
     }
 
     if (!policyItem) {
       return (
-        <EuiPageTemplate template="centeredContent">
+        <EuiPageSection grow={false} alignment="center">
```

Comment:
Same question here.
```json5
<EuiPageTemplate>
 <EuiPageTemplate.Section>
...
...
 </EuiPageTemplate.Section>
</EuiPageTemplate>

Reply:
see this comment: https://github.com/elastic/kibana/pull/162531#discussion_r1307861905